### PR TITLE
YEP 23 Express project initialisation

### PR DESCRIPTION
**Pull request by alexis-moins**
- feat(express): bootstrap project dependencies
- feat(api): added status route and ts-node-dev
- feat(api): added justfile
- fix(api): fixed autoreload in docker [wip]
- feat(api): setup docker dev environment

### CODE REVIEW

**Review by Xerataw — APPROVED** [review]
Great job, wonderful, i never saw a back well done like this before 😏

File: Makefile [hunk 1]
```diff
@@ -3,10 +3,11 @@ COLOR_RESET=\033[0m
 BBLUE=\033[1;34m
 
 # Enhanced commands
-ECHO=@ echo "${BBLUE} -->"
+ECHO=@ echo "${BBLUE}-->"
```
Comment (ASparton) [inline Makefile:6]: Thx mate :)

File: GIGZ-BACK/src/server.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const { database } = useDatabase()
+
+app.use(express.json());
+
+app.use(function(req, _, next) {
+	console.log("🚥 %s %s", req.method, req.path);
+	next();
+});
+
+app.use("/api/v1/", v1);
```
Comment (ASparton) [inline GIGZ-BACK/src/server.ts:18]: Good idea, like it

Comment (alexis-moins) [reply GIGZ-BACK/src/server.ts:18]: Thanks!

**Comment by ASparton** [comment]
Mabye it could be great to add a little Readme to explain how to run the container and open adminer etc...

**Review by ASparton — APPROVED** [review]

**Comment by alexis-moins** [comment]
Sure! I'll do that later!

**Pull request merged**